docs/pages/components/_component.vue

```diff
@@ -0,0 +1,94 @@
+<template>
+  <div>
+    <div class="hero">
+        <div class="hero__container">
+          <h1 class="title">{{ component.title }} </h1>
```

Comment:
If this were to be converted to BEM, I'd do something like `hero__title`

Reply:
These classes are not needed anymore. Will clean them up.